components/BookDetails/style.js

```diff
@@ -19,7 +20,7 @@ export const StyledFrame = styled.div`
   box-shadow: 0 0 5px rgba(0, 0, 0, 0.1);
 `;
 
-export const StyledImage = styled.image`
+export const StyledImage = styled(Image)`
```

Comment:
I would put this styled component, which is just responsible for styling the image, into a file for the "Book" component, also known as "BookImage".